src/config.ts

```diff
@@ -99,6 +104,29 @@ class Config {
     return this.WALLET_SERVICE_BASE_URL;
   }
 
+  /**
+   * Returns the base url for swap service
+   * Throws an error if it is not yet set.
+   *
+   * @return The swap service url
+   */
+  getSwapServiceBaseUrl(): string {
+    if (!this.SWAP_SERVICE_BASE_URL) {
+      throw new GetWalletServiceUrlError('Swap service base URL not set.');
```

Comment:
Should we check the network to choose an URL before failing?

Reply:
Good point. I adapted the solution already existant for `getExplorerServiceBaseUrl` and implemented it on [6a5385f](https://github.com/HathorNetwork/hathor-wallet-lib/pull/464/commits/6a5385f38ebae8d5e24022c954dc364763cded64)